src/components/pageComponents/ContentCard.js

```diff
@@ -18,14 +21,11 @@ import { Link } from "react-router-dom";
 
 const useStyles = makeStyles((theme) => ({
   card: {
-    border: "solid 2px #00817B",
     marginBottom: "16px",
   },
 
   cardContent: {
-    "&:last-child": {
-      paddingBottom: "16px",
-    },
+    paddingTop: "0px",
```

Comment:
If there is a 0px padding on top, then the attribute is redundant I think

Reply:
This acutally overrides the default 16px-padding (which looked very strange).